tests/lib/bin/TODO

```diff
@@ -0,0 +1,76 @@
+ONCE THINGS ARE RENAMED/MADE UNIFORM THIS SHOULD BECOME A README WITHOUT THE TODO PART!
+
+This directory collects useful utility commands that can be used in
+our spread test suites. (They are separate from the shell function
+libraries *.sh in the level above.)
+
+Commands here should have names that differentiate them from most
+commands in {/usr,}{/bin,/sbin} (historically we have used *-tool
+names but actually *-tool commands do exist there.)
+
+If possible the name should indicate what they do or if many things
+what their area is.
+
+All of them invoked without arguments should either do something
+informative/non-destructive or print usage.
+
+All of them should support at least a -h help option.
+
+To achieve the naming goals we are going to use names with dots in
```

Comment:
Why dots? Dashes are used more commonly and feel more natural.

Reply:
I don't find them particularly natural or unnatural.  They help with the goal of hinting that this comes from the project itself, they are not perfect because there are things using `.` in */bin, just not that many and slightlly different that the proposal. A common prefix or suffix could be another approach,  as I said many time though `-tool` is not a good one, for example samba-tool exists (and a few others). Though all the clearer ones that I could think of are more cumbersome than using dots as proposed:  tst-* testing-*  snapd-test-* ?